x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curation/curation.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { useValues, useActions } from 'kea';
+
+import { EuiPageHeader, EuiSpacer } from '@elastic/eui';
+
+import { FlashMessages } from '../../../../shared/flash_messages';
+import { SetAppSearchChrome as SetPageChrome } from '../../../../shared/kibana_chrome';
+import { BreadcrumbTrail } from '../../../../shared/kibana_chrome/generate_breadcrumbs';
+import { Loading } from '../../../../shared/loading';
+
+import { MANAGE_CURATION_TITLE } from '../constants';
+
+import { CurationLogic } from './curation_logic';
+
+interface Props {
+  curationsBreadcrumb: BreadcrumbTrail;
+}
+
+export const Curation: React.FC<Props> = ({ curationsBreadcrumb }) => {
+  const { curationId } = useParams() as { curationId: string };
+  const { loadCuration } = useActions(CurationLogic({ curationId }));
+  const { dataLoading, curation } = useValues(CurationLogic({ curationId }));
```

Comment:
To re-cap our Slack convo, I'd like to come back to this as tech debt someday to use BindLogic and potentially key CurationLogic w/ the curationId, but right now it's a bit too high of a lift for updating tests/test helpers